js/renovation/ui/scroll_view/__tests__/scrollable.test.tsx

```diff
@@ -67,6 +68,11 @@ jest.mock('../../../../core/utils/scroll_rtl_behavior', () => () => testBehavior
         expect(bottomPocket.exists()).toBe(true);
       });
 
+      it('Scrollable should have dx-scrollable-disabled if disabled', () => {
```

Comment:
"expect(instance.cssClasses).toEqual(expect.stringMatching('dx-scrollable-disabled'));" check should be in the describe('cssClasses' module

Reply:
Yep, fixed https://github.com/DevExpress/DevExtreme/pull/15636/commits/3371f7a4122918b13a658b8a49751be74f06fa71